src/app/lib/views/main_window.js

```diff
@@ -210,10 +202,15 @@
     },
 
     onAttach: function() {
-      if (os.platform() === 'win32') {
-        this.showChildView('Header', new App.View.WindowsTitleBar());
+      if (pkJson.window.frame) {
+        $('#header').remove();
+        $('#' + this.id).addClass('default-frame');
       } else {
-        this.showChildView('Header', new App.View.TitleBar());
+        if (os.platform() === 'win32') {
```

Comment:
should be "else if" :)

Reply:
From pure code styling yes, but I kept it that way to separate logic. Main if separates HTML, and then nested if includes logic for platform. I think that does improve readability. But nevertheless it can be changed.

Reply:
no big deal, do as you want ;)